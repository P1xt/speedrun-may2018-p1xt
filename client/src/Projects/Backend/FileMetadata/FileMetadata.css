.FileMetadata {
  text-align: left;
  margin-top: 40px; }
  .FileMetadata > h1 {
    text-transform: none; }
