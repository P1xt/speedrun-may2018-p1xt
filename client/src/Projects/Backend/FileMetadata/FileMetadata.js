import React from "react";
import { Container, Row, Col } from 'reactstrap';

import "./FileMetadata.css";

const FileMetadata = () => {
  return (
    <div className="FileMetadata">
      <Container>
        <Row>
          <Col>
            <h1>File Metadata</h1>
          </Col>
        </Row>
      </Container>
    </div>
  );
};

export default FileMetadata;
