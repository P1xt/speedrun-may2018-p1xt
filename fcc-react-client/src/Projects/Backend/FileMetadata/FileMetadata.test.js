import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";

import FileMetadata from "./FileMetadata";

describe("<FileMetadata />", () => {
  it("renders without crashing", () => {
    shallow(<FileMetadata />);
  });
  it("renders correctly", () => {
    const tree = renderer.create(<FileMetadata />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
