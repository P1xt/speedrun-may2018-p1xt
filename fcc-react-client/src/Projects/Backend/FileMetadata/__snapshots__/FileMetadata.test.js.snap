// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FileMetadata /> renders correctly 1`] = `
<div
  className="FileMetadata"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <h1>
          File Metadata
        </h1>
      </div>
    </div>
  </div>
</div>
`;
